src/ProxyClient/Varnish.php

```diff
@@ -1,27 +1,41 @@
 <?php
 
 /**
- * File containing the Varnish class.
- *
  * @copyright Copyright (C) eZ Systems AS. All rights reserved.
  * @license For full copyright and license information view LICENSE file distributed with this source code.
  */
+declare(strict_types=1);
+
 namespace EzSystems\PlatformHttpCacheBundle\ProxyClient;
 
-use FOS\HttpCache\ProxyClient\Varnish as FOSVarnish;
+use FOS\HttpCache\ProxyClient\Varnish as FosVarnish;
 
-/**
- * Class Varnish.
- *
- * We need to support https as Platform.sh uses https by default.
- */
-class Varnish extends FOSVarnish
+class Varnish extends FosVarnish
```

Comment:
any reason why we can't drop this?

Reply:
We could if we would like to move from `PURGE` to `PURGEKEYS` in Varnish.

Reply:
We probably should so we can align with FOS VCL as well as getting rid of the overloading, but this can be done in stages if you want.